src/lib/util.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+var EventManager = require('./eventManager');
+
+module.exports = {
+  extend: function (destination, source) {
+    for (var property in source) {
+      destination[property] = source[property];
+    }
+  },
+
+  makeEventCapable: function (destination) {
```

Comment:
Perhaps it would be better to just make the `destination.event = new EventManager()` assignment explicit inside destination? At the point wheer `makeEventCapable` is called, it is not really visible what happens.